lib/fog/ovirt/compute.rb

```diff
@@ -33,7 +33,7 @@ def initialize(client)
           @client = client
         end
 
-        # rubocop:disable Style/MethodMissing
+        # rubocop:disable Style/MethodMissingSuper, Style/MissingRespondToMissing
```

Comment:
We can rename `#respond_to?` into `#respond_to_missing?`. This will actually fix the cop, and make our code better.
See https://robots.thoughtbot.com/always-define-respond-to-missing-when-overriding

Reply:
That was already a rule of `Style/MethodMissing` which we decided not to fix originally so I would suggest fixing rubocop errors now to unblock other PRs and any refactoring to be done later in another PR as that would also require more testing.